dev_guide/cicd.adoc

```diff
@@ -0,0 +1,254 @@
+[[dev-guide-cicd]]
+= CI / CD in OpenShift
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+{product-title} This next topic will focus on continuous integration and continuous delivery (`*CI/CD*`) of applications with OpenShift.  In particular, the goal is to
+illustrate how the variety of concepts, infrastructure, hook points, and tools within OpenShift Origin facilitate `*CI/CD*`and the
```

Comment:
s/how the/how a/
Sentence runs on a bit. Consider period after "facilitate CI/CD" then a new sentence for the promotion facet.